### ISSUE TITLE
Mention related packages in README

### ISSUE THREAD

**Issue by goretkin** [goretkin/FixArgs.jl#20]
https://github.com/archermarx/PartialFunctions.jl

@archermarx , @MrBreadWater I wonder if you think it is worthwhile to combine the functionality of these two packages.

**Comment by archermarx** [comment]
Yeah it's worth considering! I need to look more at this package but I took a lot of my implementation details from how base implemented `Fix1` and so on. Would be very useful

**Comment by goretkin** [comment]
https://github.com/Tokazama/ChainedFixes.jl/blob/master/src/ChainedFixes.jl#L213-L247

**Comment by MrBreadWater** [comment]
Ooh, this would be a nice pairing. Though, I probably couldn't be of much help, because my 'contribution' to PartialFunctions consisted of 1 line a couple days ago.

**Comment by goretkin** [comment]
https://github.com/MasonProtter/LegibleLambdas.jl
https://github.com/c42f/Underscores.jl
https://github.com/tkf/UnderscoreOh.jl

**Comment by MrBreadWater** [comment]
Could be useful to combine even more of these packages, like the ones you linked, into one for a larger module. I've seen a lot of smaller modules (ie PartialFunctions, this one) aimed towards a common goal of implementing more "Functional" aspects to Julia or making it easier to use what's already in place, and all are fairly good on their own, but they might be able to work together far better if they merged.
